src/breadcrumbs.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import _ from 'lodash';
+
+export class Breadcrumbs extends React.Component {
+
+  static propTypes = {
+    className: React.PropTypes.string,
```

Comment:
We aren't using className anywhere yet, but we should. Take a look at our other components and see how we can use this.props.className.